modules/tactical/src/tactical_store.ts

```diff
@@ -124,37 +123,27 @@ export class TacticalStore implements Store {
   fetch(key: Object, version?: string): Observable<Record> {
     var recordKey: RecordKey;
     var chainKey: ChainKey = new ChainKey(key);
-    return Observable.create<Record>((observer: Rx.Observer<Record>) => {
-      if (version) {
-        recordKey = new RecordKey(version, chainKey);
-        this._idb.get(this.store + TacticalStore._recordStore, recordKey.serial)
-            .subscribe((value: Object) => {
-              if (value) {
-                observer.onNext({version: version, value: value});
-              } else {
-                observer.onNext(null);
-              }
-            }, (err: any) => { observer.onError(err); });
-      } else {
-        // fetch the most recent Record in the Chain
-        this._idb.get(this.store + TacticalStore._versionStore, chainKey.serial)
-            .subscribe((ver: Version) => {
-              if (ver && ver.version) {
-                recordKey = new RecordKey(ver.version, chainKey);
-                this._idb.get(this.store + TacticalStore._recordStore, recordKey.serial)
-                    .subscribe((value: Object) => {
-                      if (value) {
-                        observer.onNext({version: ver.version, value: value});
-                      } else {
-                        observer.onNext(null);
-                      }
-                    }, (err: any) => { observer.onError(err); });
-              } else {
-                observer.onNext(null);
-              }
-            }, (err: any) => { observer.onError(err); });
-      }
-    });
+
+    if (version) {
```

Comment:
This logic is unrelated to the commit under which it's listed. Move to a separate commit.